### PR TITLE
[Matrix] API update / compile warning fix

### PR DESCRIPTION
**Pull request by AlwinEsch**
API related to xbmc/xbmc#17438 (DEMUX_PACKET not used there)

### CODE REVIEW

**Comment by phunkyfish** [comment]
Why have an attribute hidden in all
classes if they are namespaced?

**Comment by AlwinEsch** [comment]
Was come to my mind that if another place can find it, that it must also be somewhere in code and memory listed, if now every part add it to somewhere, comes a really big list :thinking: 

Also seen some compile warnings during build.

**Comment by AlwinEsch** [comment]
Has removed this first commit about attribute hidden and pushed to check travis build about

**Comment by phunkyfish** [comment]
Remind me again why the bump to 7.0.0 is needed?

**Comment by AlwinEsch** [comment]
> Remind me again why the bump to 7.0.0 is needed?

Is about the inputstream API where used inside PVR. By inputstream is to have them correct in "C" and few memleaks fixed there.
Problem is only that PVR has this stream parts also in use and need increase too.

WIth major increase is related to @ksooo (and he arranged by Team) where want it by API change.

**Comment by phunkyfish** [comment]
Ok, that’s good with me.

File: pvr.iptvsimple/addon.xml.in [hunk 1]
```diff
@@ -169,34 +169,9 @@
...
-
-v6.0.1
-- Update PVR API 7.0.1
-
-v6.0.0
-- Update PVR API 7.0.0
-- Rework addon to support new API interface
+v7.0.0
+- Update: PVR API 7.0.2
+- Fixed: Prevent some compile warnings by add of ATTRIBUTE_HIDDEN
```
Comment (phunkyfish) [inline pvr.iptvsimple/addon.xml.in]: Like should be removed

File: pvr.iptvsimple/changelog.txt [hunk 2]
```diff
@@ -1,3 +1,7 @@
+v7.0.0
+- Update: PVR API 7.0.2
+- Fixed: Prevent some compile warnings by add of ATTRIBUTE_HIDDEN
```
Comment (phunkyfish) [inline pvr.iptvsimple/changelog.txt]: Line should be removed.

**Review by phunkyfish — CHANGES_REQUESTED** [review]

**Review by phunkyfish — APPROVED** [review]

**Pull request merged**